src/sqlfluff/core/rules/config_info.py

```diff
@@ -103,6 +103,13 @@
             "Should alias have an explict AS or is implicit aliasing required?"
         ),
     },
+    "group_by_and_order_by_style": {
+        "validation": ["consistent", "implicit", "explicit"],
+        "definition": (
+            "Should GROUP BY/ORDER BY require explict column references "
+            "or are implicit positional column references required?"
```

Comment:
Should we make this more consistent with this:

```
    "single_table_references": {
        "validation": ["consistent", "qualified", "unqualified"],
        "definition": "The expectation for references in single-table select",
    },
```

```suggestion
            "The expectation for using explicit column name references "
            "or implicit position references"
```

Also is `implicit`/`explicit` the right values? Should it be `names`/`positions`?

Reply:
So I think either are fine, you explicitly refer to the columns by name or implicitly by postition. Could be swayed either way if you have a strong preference?

Reply:
OK I'm fine with names as is. Do think we should change comment though 1) to be more consistent and 2) as kinda of covers consistent use case more.

Reply:
I've updateed the message in the latest commit